pkg/controller/cassandracluster/pod_operation.go

```diff
@@ -677,18 +677,21 @@ func (rcc *ReconcileCassandraCluster) runUpgradeSSTables(hostName string, cc *ap
 
 func (rcc *ReconcileCassandraCluster) runRebuild(hostName string, cc *api.CassandraCluster, dcRackName string, pod v1.Pod) error {
 	var err error
+	var keyspaces []string
 	var rebuildFrom, labelSet = pod.GetLabels()["operation-argument"]
 	operation := strings.Title(api.OperationRebuild)
 
 	logrus.WithFields(logrus.Fields{"cluster": cc.Name, "rack": dcRackName, "pod": pod.Name,
 		"hostName": hostName, "operation": operation}).Info("Operation start")
 
+	jolokiaClient, err := NewJolokiaClient(hostName, JolokiaPort, rcc,
+		cc.Spec.ImageJolokiaSecret, cc.Namespace)
+
 	if labelSet != true {
 		err = errors.New("operation-argument is needed to get the datacenter name to rebuild from")
+	} else if keyspaces, err = jolokiaClient.HasDataInDC(rebuildFrom); err == nil  && len(keyspaces) == 0 {
+		err = fmt.Errorf("%s is not has no nonLocalKeyspace defined", rebuildFrom)
```

Comment:
2 things : 
- `HasDataInDC` along with its comments should be renamed. It returns a list of keyspaces found at the new current Jolokia server that have data replicated to the provided DC
- `"%s is not has no nonLocalKeyspace defined"` -> `"%s has no keyspace to replicate data from`